packages/types/src/sszTypes.ts

```diff
@@ -8,7 +8,7 @@ import {ssz as electra} from "./electra/index.js";
 import {ssz as phase0} from "./phase0/index.js";
 
 export * from "./primitive/sszTypes.js";
-export {phase0, altair, bellatrix, capella, deneb, electra};
+export {phase0, altair, bellatrix, capella, deneb, electra, electra as fulu};
```

Comment:
can we just define empty types for fulu for now? might be better than this

Reply:
It's not possible to do that.  We have `ssz[fork].SignedBeaconBlock` and other types all over the codebase.  Must have a full set of types that match the signature in order for typescript to be happy with indexing `ssz` with `fork`